mrbean/src/test/java/com/fasterxml/jackson/module/mrbean/TestAbstractClasses.java

```diff
@@ -16,11 +16,15 @@
         int y;
 
         protected Bean() { }
-        
+
         public abstract String getX();
 
         public String getFoo() { return "Foo!"; }
         public void setY(int value) { y = value; }
+
+        // also verify non-public methods
+        protected abstract String getZ();
```

Comment:
Not sure if we are expected to implement non-public getters... although probably makes sense (setters def. need to be implemented).

Reply:
I wasn't sure whether non-public methods were supported until I wrote the test, which I decided to do after exploring the differences between `getMethod` and `getDeclaredMethod`.  Once I realized that non-public methods were working the same as public ones, I figured it was worthwhile to create a test to verify that behavior wasn't regressed, per [Hyrum's law](https://www.hyrumslaw.com/).